dll/win32/shell32/utils.cpp

```diff
@@ -796,6 +796,64 @@ SHCreatePropertyBag(_In_ REFIID riid, _Out_ void **ppvObj)
     return SHCreatePropertyBagOnMemory(STGM_READWRITE, riid, ppvObj);
 }
 
+/*************************************************************************
+ *  SHSetUnreadMailCountW [SHELL32.336]
+ *
+ * @see https://learn.microsoft.com/en-us/windows/win32/api/shellapi/nf-shellapi-shsetunreadmailcountw
+ */
+EXTERN_C
+HRESULT WINAPI
+SHSetUnreadMailCountW(
+    _In_ PCWSTR pszMailAddress,
+    _In_ DWORD dwCount,
+    _In_ PCWSTR pszShellExecuteCommand)
+{
+    CString strKey = L"Software\\Microsoft\\Windows\\CurrentVersion\\UnreadMail";
+    strKey += L'\\';
+    strKey += pszMailAddress;
```

Comment:
You could just have added the `\\` in the first string ^^

Reply:
https://github.com/reactos/reactos/commit/0bb8b46ba8e5c25c95a4250edbac8863523b8608